sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -20,6 +20,26 @@ and includes six main functions:
 - [Azure Subscription][azure_subscription]
 - [Cognitive Services or Text Analytics account][text_analytics_account] to use this package.
 
+### Include the Package
+
+[//]: # ({x-version-update-start;com.azure:azure-ai-textanalytics;current})
+```xml
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-ai-textanalytics</artifactId>
+    <version>5.1.0</version>
+</dependency>
+```
+[//]: # ({x-version-update-end})
+**Note:** This version of the client library defaults to the `v3.1` version of the service.
+
+This table shows the relationship between SDK services and supported API versions of the service:
+
+|SDK version|Supported API version of service
+|-|-
+|5.0.x | 3.0
+|5.1.x | 3.0, 3.1 (default)
```

Comment:
maybe just me, but I feel like the more recent release should be at the top of the table.